app/views/wiki/edit.html.erb

```diff
@@ -26,11 +26,13 @@
 
 <div class="col-sm-9">
 
-  <% if @node && @node.path
-    url = { :controller => "wiki", :action => "update", :id => @node.id }
-  else 
-    url = { :controller => "wiki", :action => "create", :id => params[:id] }
-   end %>
+  <%
+    if @node.persisted?
```

Comment:
Here, we may still want to check that `@node` exists before calling a sub-method. This can occasionally cause page failures as a pattern, though I'm not sure it would in this case? I do so by habit these days.